packages/containers/src/cmfModule.js

```diff
@@ -0,0 +1,16 @@
+import { cmfConnect } from '@talend/react-cmf';
+import omit from 'lodash/omit';
+import * as allComponents from '@talend/react-components';
+import * as containers from './index';
+
+const components = Object.keys(allComponents).reduce((acc, key) => {
+	if (!acc[key] && typeof allComponents[key] === 'function') {
+		// eslint-disable-next-line no-param-reassign
+		acc[key] = cmfConnect({})(allComponents[key]);
+	}
+	return acc;
+}, omit(containers, ['actionAPI']));
```

Comment:
I know it's not part of this PR, but I don't think this utility have something to do in the current package. It seems to belong to cmf package. Maybe it could be fix in an other PR or I open a JIRA issue ?

Reply:
There two action API, one for the CMF already which I would like to remove.
In Container you have multiple Action(s/dropdown/...) containers which have in some way the same mapStateToProps, so we have moved that code here. For Me it should not be exposed at all but it s too late.

So now I remove this object from the import, and this is why I would like to not put the cmfModule at the index.js

So we have one other solution it s to put cmfModule as a default export of containers